### PR TITLE
FIX: BugSnag would cycle through sendEvents in offline mode

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xh0h9b75ehhzhm22bny46tq654?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/8008)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xzp0dh44cveed5gcxfz7va8ah8?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/8011)

File: blue_modules/BlueElectrum.js [hunk 1]
```diff
@@ -313,6 +313,7 @@ async function getRandomDynamicPeer() {
  * @returns {Promise<Object>}
  */
 module.exports.getBalanceByAddress = async function (address) {
+  if (await isDisabled()) return;
```
Comment (Overtorment) [inline blue_modules/BlueElectrum.js:316]: that feels wrong. in theory execution should not reach here in offline mode

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/1nmpavvwdfrvkvm7fvjyzdktkm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/8023)

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**